### PR TITLE
✨ feat: Add Startup Probe to Healthcheck Middleware

### PR DESCRIPTION
**Pull request by kirankumar-grootan**
## Description

Added default startup probe route for fiber v3 

https://github.com/gofiber/fiber/issues/3068

### CODE REVIEW

**Comment by ReneWerner87** [comment]
why did you close it? @kirankumar-grootan

**Comment by ReneWerner87** [comment]
@kirankumar-grootan 
i see the healthcheck mw is better in v3 https://docs.gofiber.io/next/middleware/healthcheck

can you still add the default path for the startup probe and documentation so that people don't wonder why we didn't include them

**Comment by kirankumar-grootan** [comment]
@ReneWerner87 I have add the startup default endpoint and updated docs and test accordingly.

**Comment by ReneWerner87** [comment]
Will check it tomorrow, thx

File: middleware/healthcheck/healthcheck_test.go [hunk 1]
```diff
@@ -51,13 +55,17 @@ func Test_HealthCheck_Default(t *testing.T) {
 	app := fiber.New()
 	app.Get("/livez", NewHealthChecker())
 	app.Get("/readyz", NewHealthChecker())
+	app.Get(DefaultStartupEndpoint, NewHealthChecker())
```
Comment (luk3skyw4lker) [inline middleware/healthcheck/healthcheck_test.go:58]: I think that maybe to have everything padronized the other paths should use the default variable too or this path should use a string path just like the others (I would personally prefer the first option, just didn't remembered it when I wrote the middleware).

What do you think @ReneWerner87?

**Comment by gaby** [comment]
@coderabbitai full review

File: middleware/healthcheck/healthcheck_test.go (hunk 1 above)
Comment (ReneWerner87) [reply middleware/healthcheck/healthcheck_test.go:58]: @kirankumar-grootan yeah pls use the default var everywhere in ths test

Comment (kirankumar-grootan) [reply middleware/healthcheck/healthcheck_test.go:58]: @ReneWerner87  changed from string to use the const in test cases

**Review by luk3skyw4lker — APPROVED** [review]

**Review by gaby — APPROVED** [review]
LGTM

File: docs/middleware/healthcheck.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ id: healthcheck
 
 # Health Check
```
Comment (efectn) [inline docs/middleware/healthcheck.md:6]: Can you update v3 what's new docs as well

Comment (kirankumar-grootan) [reply docs/middleware/healthcheck.md:6]: @efectn I have added the information and migration changes in whats_new.md file

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**